docs/src/main/asciidoc/neo4j-guide.adoc

```diff
@@ -0,0 +1,545 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+:neo4j_version: 3.5.6
+= Quarkus - Neo4j
```

Comment:
The title has to be the first element after the comment.

Reply:
You marked it as resolved but the title is still after the attribute. It needs to be before the attribute (otherwise it's not properly considered as a title on the website).